### PR TITLE
Fluent theme: Add TeachingBubble styles

### PR DESCRIPTION
**Pull request by mikewheaton**
#### Pull request checklist

- [x] Addresses an existing issue: #7138
- [x] Include a change request file using `$ npm run change`

#### Description of changes
Adds styles for TeachingBubble to the Fluent theme.

#### Before
image
image

#### After
image
image

### CODE REVIEW

**Comment by Jahnp** [comment]
Hey @mikewheaton , it looks like the deploy link isn't working. Would you mind sharing before/after screens so we can get a sense of what the changes look like?

**Comment by KevinTCoughlin** [comment]
> 
> 
> Hey @mikewheaton , it looks like the deploy link isn't working. Would you mind sharing before/after screens so we can get a sense of what the changes look like?

+1 to this, testing locally I mostly see the trigger button change when the Theme is Fluent

image

File: packages/fluent-theme/src/fluent/styles/TeachingBubble.styles.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      selectors: {
+        '&:hover': {
+          color: palette.black
+        }
+      }
+    },
+    subComponentStyles: {
+      callout: {
+        root: {
+          boxShadow: Depths.depth16
```
Comment (Vitalius1) [inline packages/fluent-theme/src/fluent/styles/TeachingBubble.styles.ts]: Looks weird to me that before this was:
https://github.com/OfficeDev/office-ui-fabric-react/blob/5bc73a304b843bf183bf77659e0165d395f4af91/packages/office-ui-fabric-react/src/components/TeachingBubble/TeachingBubble.styles.ts#L91
and now from nowhere `depth16`, but designers know better 🙄 .

**Comment by mikewheaton** [comment]
I've added before/after screenshots for future reference.

**Pull request merged**